src/Symfony/Bridge/PhpUnit/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2015-2015 Fabien Potencier
```

Comment:
Technically, the code was written in 2014. so, the range should be 2014-2015, which would also avoid any further PR to "fix" it to just 2015.